### PR TITLE
Category filter

### PR DESCRIPTION
**Pull request by dianashk**
Add `categories` terms filter.

Connected to pelias/pelias#150

### CODE REVIEW

**Comment by orangejulius** [comment]
I'd really love to see this branch be rebased on top of master to get rid of the merge commit and keep the history cleaner. A lot has happened since the first commit.

**Comment by orangejulius** [comment]
Also, do we want to keep version bumps out of feature branches? If we bump versions in feature branches, then we run into issues when branches are merged close to one another.

**Comment by dianashk** [comment]
@orangejulius, I'll need a little help to do the rebase correctly. The version bumping is always a point of contention. You're right that it might change between now and when I actually merge it. I guess this is the optimist in me thinking it will be a short interval. :)

**Comment by missinglink** [comment]
:+1: very nice

**Comment by trescube** [comment]
I can't help with any of the git things, but :shipit:

**Comment by orangejulius** [comment]
oh, i am happy to help with the rebase sometime, just let me know!

File: view/categories.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  // base view
+  var view = { terms: {} };
+
+  var value = vs.var('input:categories').get();
+  if (!(value instanceof Array)) {
+    value = [ value ];
+  }
+
+  // terms query
+  view.terms[ vs.var('categories:field') ] = value;
```
Comment (orangejulius) [inline view/categories.js:18]: There is a [terms view](https://github.com/pelias/query/blob/master/view/terms.js) now!

Comment (orangejulius) [inline view/categories.js:18]: Although it's so tiny I'm not sure it even makes sense to use here. But consolidating functionality could still be nice.

**Comment by orangejulius** [comment]
LGTM

File: view/categories.js (hunk 1 above)
Comment (dianashk) [inline view/categories.js:18]: I'm not exactly sure how to apply that here, so I'm happy to not change it at this time. Thanks for calling it out, as I kinda forgot about that thing.

Comment (orangejulius) [inline view/categories.js:18]: An example of how to use it would be the [layers view](https://github.com/pelias/query/blob/master/view/layers.js)

**Pull request merged**